src/routes/plans/[id]/+page.svelte

```diff
@@ -104,7 +107,26 @@
     $view = { ...data.initialView };
   }
 
-  $: $activitiesMap = createActivitiesMap($plan, $activityDirectives, $simulationSpans);
+  $: if ($plan) {
+    effects
+      .getResourcesExternal($plan.id, $plan.start_time, $plan.duration)
+      .then(newResources => ($externalResources = newResources));
+  }
+
+  $: if ($plan && $simulationDataset !== undefined) {
+    if ($simulationDataset !== null) {
+      const datasetId = $simulationDataset.id;
+      effects
+        .getResources(datasetId, $plan.start_time, $plan.duration)
```

Comment:
What's the difference between `getResources` and `getResourcesExternal`?

Reply:
`getResources` just gets resources (simulation results) from a regular Aerie simulation.
`getResourcesExternal` gets resources (also simulation results) that happen outside Aerie. Typically this is used to visualize resources from other modeling tools. See [these docs](https://nasa-ammos.github.io/aerie-docs/planning/external-datasets/).